core/src/main/java/org/stellar/anchor/sep24/Sep24RefundPayment.java

```diff
@@ -1,11 +1,33 @@
 package org.stellar.anchor.sep24;
 
 public interface Sep24RefundPayment {
+  /**
+   * The payment ID that can be used to identify the refund payment. This is either a Stellar
+   * transaction hash or an off-chain payment identifier, such as a reference number provided to the
+   * user when the refund was initiated. This id is not guaranteed to be unique.
```

Comment:
❓   Why it's not guaranteed to be unique? Is it because we could have multiple payments in the same tx?

Reply:
I cannot figure out either. I posted a message in Slack.